src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/experimental/CalcMetadata.java

```diff
@@ -0,0 +1,36 @@
+package org.broadinstitute.hellbender.tools.spark.sv.evidence.experimental;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.broadinstitute.barclay.argparser.BetaFeature;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.programgroups.StructuralVariationSparkProgramGroup;
+import org.broadinstitute.hellbender.engine.spark.GATKSparkTool;
+import org.broadinstitute.hellbender.tools.spark.sv.StructuralVariationDiscoveryArgumentCollection;
+import org.broadinstitute.hellbender.tools.spark.sv.evidence.ReadMetadata;
+import org.broadinstitute.hellbender.tools.spark.sv.evidence.SVReadFilter;
+
+import java.util.Collections;
+
+@CommandLineProgramProperties(summary="Dump some data about the reads.",
+        oneLineSummary="Dump some data about the reads.",
+        programGroup=StructuralVariationSparkProgramGroup.class)
+@BetaFeature
+public class CalcMetadata extends GATKSparkTool {
```

Comment:
I know this is only for testing / interactive debugging right now, but it would be great to flesh this out into a full tool that calculates and writes out read metadata and then provide a `readMetadata` method that can parse the output file into a full `ReadMetadata` object. That would let us write and/or downstream tools that would like to have the read metadata but not have to recompute it from the reads. Could you maybe add a ticket to do that in the future?

Reply:
I hacked in a solution that uses Kryo serialization to dump a file that could be deserialized by some other tool.  Better than nuthin'.